bcrypt/default_test.go

```diff
@@ -0,0 +1,21 @@
+package bcrypt
+
+import "testing"
+
+func TestBcryptVerifyPassword(t *testing.T) {
```

Comment:
This test proves that your hasher works, but it does not mean that it will work with Django hasher.

You need to generate a hash password from DJango and add a test for it.

Reply:
ook, will do!

Reply:
Thanks :)